docs/src/release-notes-js.md

```diff
@@ -6,6 +6,76 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.43
+
+### New APIs
+
+- Method [`method: BrowserContext.clearCookies`] now supports filters to remove only some cookies.
+
+  ```js
+  // Clear all cookies.
+  await context.clearCookies();
+  // New: clear cookies with a particular name.
+  await context.clearCookies({ name: 'session-id' });
+  // New: clear cookies for a particular domain.
+  await context.clearCookies({ domain: 'my-origin.com' });
+  ```
+
+- New trace mode `retain-on-first-failure`. In this mode, trace is recorded for the first run of each test, but not for retires. When test run failes, the trace file is retained, otherwise it is removed.
```

Comment:
```suggestion
- New trace mode `retain-on-first-failure`. In this mode, trace is recorded for the first run of each test, but not for retires. When test run fails, the trace file is retained, otherwise it is removed.
```